### PR TITLE
metrics(ticdc): split mysql sink row, and refine sink metrics.

### LINKED ISSUE #5013: add a row for MySQL

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #5013, #4964 

### What is changed and how it works?

1. add MySQL sink row
image

2. move some panel to Kafka sink row
image

5. refine sink related metrics

split statistics metrics, such as `write duration` and `row count` by sink type, only `DB` and `MQ` at the moment.

### Check List 

Tests 

Code changes

Side effects

Related changes

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: cdc/sink/statistics.go [hunk 1]
```diff
@@ -149,7 +167,7 @@ func (b *Statistics) PrintStatus(ctx context.Context) {
 	atomic.StoreUint64(&b.totalDDLCount, 0)
 
 	log.Info("sink replication status",
-		zap.String("name", b.name),
+		zap.Any("sinkType", b.sinkType),
```
Comment (overvenus) [inline cdc/sink/statistics.go:170]:
```suggestion
		zap.Stringer("sinkType", b.sinkType),
```

File: cdc/sink/buffer_sink.go [hunk 2]
```diff
@@ -61,17 +61,13 @@ func newBufferSink(
...
-	metricEmitRowDuration prometheus.Observer
-	metricTotalRows       prometheus.Counter
+	metricTotalRows prometheus.Counter
 }
 
 func (b *bufferSink) run(ctx context.Context, errCh chan error) {
 	changefeedID := util.ChangefeedIDFromCtx(ctx)
 	state := runState{
-		metricFlushDuration:   flushRowChangedDuration.WithLabelValues(changefeedID, "Flush"),
-		metricEmitRowDuration: flushRowChangedDuration.WithLabelValues(changefeedID, "EmitRow"),
```
Comment (overvenus) [inline cdc/sink/buffer_sink.go:73]: Can we remove the `flushRowChangedDuration` in metrics.go?

**Review by overvenus — APPROVED** [review]

**Comment by overvenus** [comment]
Please add labels.

File: cdc/sink/buffer_sink.go [hunk 3]
```diff
@@ -154,12 +146,11 @@ func (b *bufferSink) runOnce(ctx context.Context, state *runState) (bool, error)
 		}
 		b.tableCheckpointTsMap.Store(tableID, checkpointTs)
 	}
-	now := time.Now()
-	state.metricFlushDuration.Observe(now.Sub(startFlush).Seconds())
-	if now.Sub(startEmit) > time.Second {
+	elapsed := time.Since(start)
+	if time.Since(start) > time.Second {
```
Comment (hi-rustin) [inline cdc/sink/buffer_sink.go:150]: Why not use elapsed here?

File: cdc/sink/buffer_sink.go [hunk 4]
```diff
@@ -61,21 +61,15 @@ func newBufferSink(
 type runState struct {
 	batch [maxFlushBatchSize]flushMsg
 
-	metricFlushDuration   prometheus.Observer
-	metricEmitRowDuration prometheus.Observer
```
Comment (hi-rustin) [inline cdc/sink/buffer_sink.go:65]: Why do we need to delete these two items?

Comment (3AceShowHand) [reply cdc/sink/buffer_sink.go:65]: It's removed from monitoring and no need to track these metrics. Most of the time, these 2 do not cost too much time, might have some spikes, but do not have significant information, so removed.

**Review by sdojjy — APPROVED** [review]

**Pull request merged**